.readthedocs.yaml

```
@@ -4,6 +4,30 @@
 
 version: 2
 
+build:
+  os: "ubuntu-22.04"
+  apt_packages:
+    - automake
+    - libtool
+    - make
+    - g++
+    - libqd-dev
+    - libqhull-dev
+    - libmumps-seq-dev
+    - liblapack-dev
+    - libopenblas-dev
+    - libpython3-dev
+    - ufraw
```

https://stackoverflow.com/questions/52998331/imagemagick-security-policy-pdf-blocking-conversion
```suggestion
    - dcraw
```